src/features/tags/l10n/messageIds.ts

```diff
@@ -15,7 +15,7 @@ export default makeMessages('feat.tags', {
     groupCreatePrompt: m<{ groupName: string }>('Add "{groupName}"'),
     groupLabel: m('Group'),
     groupSelectPlaceholder: m('Type to search or create a group'),
-    submitCreateTagButton: m('Create and apply'),
+    submitCreateTagButton: m('Create'),
```

Comment:
This hasn't gone through on the dev server, which is probably because the original value is still in the YAML files. Maybe we can replace this with a new key?

```suggestion
    createTagButton: m('Create'),
```